### DISCUSSION TITLE
Redeploy rabbitmq cluster

### DISCUSSION THREAD

**Discussion by TimJongerius** [rabbitmq/cluster-operator#1142] [Q&A]
Hi,

I've tried to recover my rabbitmq cluster after wiping the entire K8s by reclaiming the volumes and redeploying the operator and my rabbitmqcluster resource. I can see that the PVs get mounted to the corresponding pods as they were before. But when I'm logging into the management console all configurations are lost. I guess the configuration is saved elsewhere. Is there a way to also recover the configuration? I need this because my customer is running rabbitmq in a staging environment which gets deleted regularly for cost reasons and then redeployed with IaC on demand.

Thanks for your help!

**Comment by ChunyiLyu** [comment]
Hi @TimJongerius, by configurations do you mean topology objects like queues, bindings, and exchanges (excluding messages)? They should be stored on disk and the previous PV should have the information. Instead of reattaching PVs, you can try saving configurations and/or pre-configure clusters with export and import definitions: https://www.rabbitmq.com/definitions.html#:~:text=Definitions%20can%20be%20exported%20to,own%20replica%20of%20all%20definitions. Does this fit your use case?
